modules/apps_api/app/workers/apps_api/fetch_connections.rb

```diff
@@ -1,38 +0,0 @@
-# frozen_string_literal: true
-
-require 'sidekiq'
-require_relative '../../../lib/apps_api/notification_service'
-
-module AppsApi
-  class FetchConnections
-    include Sidekiq::Worker
-    include SentryLogging
-    sidekiq_options unique_for: 24.hours
-
-    def perform
-      return if cancelled?
-
-      notif_service = AppsApi::NotificationService.new
-
-      # process all disconnection events
-      notif_service.handle_event(
-        'app.oauth2.as.consent.revoke',
-        Settings.vanotify.services.lighthouse.template_id.disconnection_template
```

Comment:
I think these settings entries for both connection & disconnection templates can now be removed.